docs/source/traitsui_user_manual/handler.rst

```diff
@@ -45,21 +45,121 @@ means of the UIInfo object.
 
 Whenever TraitsUI creates a window or panel from a View, a UIInfo object is
 created to act as the Handler's reference to that window and to the objects
-whose :term:`trait attribute`\ s are displayed in it. Each entry in the View's
-context (see :ref:`the-view-context`) becomes an attribute of the UIInfo
-object. [12]_ For example, the UIInfo object created in
-:ref:`Example 7 <example-7-using-a-multi-object-view-with-a-context>`
+whose :term:`trait attribute`\ s are displayed in it.  This object holds a
+reference to the UI instance in it's **ui** trait, and whether the or not
```

Comment:
"it's" -> "its"

Reply:
there's also a spurious "the" between "whether" and "or".

BTW, sorry for not grouping the prior comments together in a review; I'll do that from this point onwards.